src/hooks/node/useNodeStatus.ts

```diff
@@ -1,6 +1,15 @@
 import { GetStatusResponse } from '@ironfish/sdk'
 import { useEffect, useState } from 'react'
 
+interface NodeStatusResponse extends GetStatusResponse {
+  node: {
+    status: 'started' | 'stopped' | 'error'
+    version: string
+    git: string
+    nodeName: 'My Node Name Example'
+  }
+}
+
```

Comment:
Do we need this in feature?

Reply:
Not sure but there is no nodeName in GetStatusResponse, will add a comment

Reply:
https://github.com/iron-fish/ironfish/blob/f1dcebb99484d5a1dce940eedf0f32918699edd2/ironfish/src/rpc/routes/node/getStatus.ts#L20
Possibly you are not update dependencies